src/actions/referralsActions.js

```diff
@@ -82,17 +118,37 @@ export const sendReferralInvitationsAction = (invitations: ReferralInvitation[])
           token: token.token,
         });
 
-        await api.sendReferralInvitation({
+        const { error } = await api.sendReferralInvitation({
           token: token.token,
           walletId,
           referralLink,
           email,
           phone,
         });
+
+        if (error) {
+          const errorMessage = get(error, 'response.data.message');
+          inviteErrorAction(dispatch, errorMessage);
+        } else {
+          let updatedInvitationCount = count + invitationContacts.length;
+          const currentDate = new Date().toJSON().slice(0, 10);
+          if (date !== currentDate) updatedInvitationCount = invitationContacts.length;
+          const updatedAlreadyInvitedContacts = [...alreadyInvitedContacts, ...invitationContacts];
+          await dispatch(saveDbAction('referralData', {
+            referrals: {
+              alreadyInvitedContacts: updatedAlreadyInvitedContacts,
+              sentInvitationsCount: { count: updatedInvitationCount, date: currentDate },
+            },
+          }));
```

Comment:
I think we can just store this on the redux state, and the persist it.

Reply:
Hm, I've followed our usual way of persisting data - storing it to local db using saveDbAction.
Or I got you wrong?

Also, this part should be reworked once we will get available invitation count form backend :)